packages/lesswrong/lib/collections/featuredResources/newSchema.ts

```diff
@@ -72,7 +72,7 @@ const schema = {
   expiresAt: {
     database: {
       type: "TIMESTAMPTZ",
-      nullable: true,
+      nullable: false,
```

Comment:
This was `NOT NULL` in the actual database for us, but we only have two records.  I'm not actually sure it's supposed be `NOT NULL`; it was marked `nullable: true` in the old schema so this might be wrong and I'll need to undo this part if the EA forum has any values in here that are null.